src/MonitoringFunctions.Common/DataService/Providers/KustoDataService.cs

```diff
@@ -13,13 +13,19 @@ namespace MonitoringFunctions
     internal sealed class KustoDataService : IDataService
     {
         private const string ServiceNameAndRegion = "dotnetinstallcluster.eastus2";
-        private const string DatabaseName = "dotnet_install_monitoring_database";
+        private static readonly string? DatabaseName = Environment.GetEnvironmentVariable("kusto_db_name");
 
         private readonly KustoTable<HttpRequestLogEntry> _httpRequestLogsTable;
         private readonly KustoTable<ScriptExecutionLogEntry> _scriptExecutionLogsTable;
 
         internal KustoDataService()
         {
+            if (string.IsNullOrWhiteSpace(DatabaseName))
+            {
+                throw new ArgumentException($"{nameof(DatabaseName)} was not correctly configured. " +
```

Comment:
nit: Is `ArgumentException` appropriate here, since `DatabaseName` is not argument of constructor? Maybe use instead `InvalidOperationException`, citing documentation:
> The exception that is thrown when a method call is invalid for the object's current state.